docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md

```diff
@@ -403,7 +403,7 @@ The following code examples use a Databricks SQL connection string. A connection
 The following code is an example of a Databricks SQL connection string format:
 
 ```python
-my_connection_string = "databricks+connector://token:<token>@<host>:<port>/<database>?http_path=<http_path>"
+my_connection_string = f"databricks://token:{token}@{host}:{port}/{database}?http_path={http_path}"
```

Comment:
I agree that this is a better format but do our other examples don't use `f` strings here?

Reply:
Did you guys see my other comment in #documentation channel? You need to specify the catalog and schema as named params or GX won't be able to find tables when you `add_table_asset()`. 

`f"databricks://token:{token}@{host}:{port}/?http_path={http_path}&catalog={catalog}&schema={schema}"`